dali/base/dadfs.cpp

```diff
@@ -1243,13 +1243,11 @@ static void setUserDescriptor(Linked<IUserDescriptor> &udesc,IUserDescriptor *us
     udesc.set(user);
 }
 
+static bool scopePermissionsAvail = true;
```

Comment:
Although I like the idea of disallowing others from changing this variable, I think it's beyond the scope of this PR.   Also, the pattern throughout this C style file is to declare them static with file scope